lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -1144,4 +1146,44 @@ module.exports = function(self, options) {
     }
   };
 
+  self.addFixOrientationsMigration = function() {
+    self.apos.migrations.add(self.__meta.name + '.fixOrientations', function(callback) {
+      return self.apos.migrations.each(
+        self.db,
+        { group: 'images' },
+        5,
+        function(attachment, callback) {
```

Comment:
Do not take a callback, just return a promise (mongodb methods return promises if not given a callback, too, so this will be easy to do).

Reply:
`each` iterators are permitted to return a promise, in which case they should not take `callback` or invoke it.

Reply:
removed